proxy/installer/squid.conf

```
@@ -24,6 +24,7 @@ memory_replacement_policy heap GDSF
 
 # cache repodata only few minutes and then query parent whether it is fresh
 refresh_pattern /XMLRPC/GET-REQ/.*/repodata/.*$ 0 1% 1440 reload-into-ims refresh-ims
+refresh_pattern /ks/.*/repomd.xml 0 1% 1440 ignore-no-cache reload-into-ims refresh-ims
```

Why not including the entire `repodata` directory here? As we do for other repository paths here.

With your current proposal only `repomd.xml` file will be refreshed but not other metadata files. Not sure if I'm missing something.

I would suggest:

```suggestion
refresh_pattern /ks/.*/repodata/.*$ 0 1% 1440 ignore-no-cache reload-into-ims refresh-ims
```

---

> Why not including the entire `repodata` directory here? As we do for other repository paths here.
> [...]
> I would suggest:
> [...]

I'm fine with your suggestion - my initial approach is usually to be the least "intrusive" and since I noticed the problem only for repomd.xml, I used the most limiting pattern.

---

I am not expert here. I just went with the suggestion from Jens here. Thus I will just apply your suggestion.